packages/manager/modules/vps/src/additional-disk/vps-additional-disk.controller.js

```diff
@@ -26,6 +28,14 @@ export default class {
     this.hasAdditionalDisk();
   }
 
+  static getDiskStateInfo({ state }) {
+    return {
+      error: VPS_DISK_STATES.ERROR.includes(state),
```

Comment:
you can use state === VPS_DISK_STATUS.DISCONNECTED

Reply:
related to this: 
https://github.com/ovh/manager/pull/6417#discussion_r803550327